kernel/Kconfig

```diff
@@ -148,6 +148,7 @@ config SCHED_CPU_MASK
 config MAIN_STACK_SIZE
 	int "Size of stack for initialization and main thread"
 	default 2048 if COVERAGE_GCOV
+	default 1024 if X86
```

Comment:
why this change?
commit message described we're fixing only ARM
if you have an x86 fix, please link GH issue and post as a separate commit in this PR